### ISSUE TITLE
Feature request : More custom widget 

### ISSUE THREAD

**Issue by angus2064** [faush01/plugin.video.embycon#69]
Hello ! I use EMBYCON with Embuary and I really love to use these together ,but that would be awesome to get more "custom widget" included with your add-on. 
Exemple ,Having a widget "random movie based on genre" would be awesome. 
 Or a variant of the widget "because you watched : " but with chosen a movie in the 3 last seeing. 
And having those on tv-show too would be great. 

These widget are already include in Embuary but not works for EMBYCON. 

If you could include these change in your next update that would be perfect.

**Comment by faush01** [comment]
These sound interesting.

Would you expect these to be integrated into the Embuary skin automatically? or you would select them from the widget node in the main EmbyCon add-on menu?

**Comment by angus2064** [comment]
If these could be integrated into the Embuary skin (in the future) that would be even better for newbie users. But anyway ,Embuary skin let us parameter "personalized widget" ,we just need to indicate where is located the widget target (for exemple ,I created a widget from Embycon add-on menu under "custom widget" "Random movies") so if we can select them from the widget node in the main EmbyCon add-on menu ,we will be able to "create" a personalized widget usable anywhere in Embuary skin. 

Thanks for your answer 😉

**Comment by faush01** [comment]
I will add some new items to the custom widget entry for now, perhaps in the future the skin dev can work their magic to add them.

 - X number of random movie based on genre
 - X number of movies because you watched (random from last 5 watched movies)

Not sure how to handle the TV Shows yet, suggest episodes or suggest TV Shows

**Comment by angus2064** [comment]
Perfect !! That's a very good news 👍
About TV Shows ,I personally think than "suggest TVshow" will be enough.

**Comment by faush01** [comment]
I have added Movie recommendations to the Custom Widget list. (Beta 1.9.39)

I have to get creative with this, when you ask for recommendations Emby gives you sets of recommendations based on list of things you have watched, I did not want a list of a list so I had to sub sample the returned lists to give just the one list with random samples taken from all the suggestions. This looks like it works ok and gives you a recommendation list basked on the last few movies you watched.

This is for movies only as I dont see any API for TV Show recomendatiosn at all so can not to TV Shows.
Also recommendations dont filter on genre so can not give recommendations for each genre.

Try out the new item in the custom widget list and see if it works for you and we can take it from there as to how or what we add next.
